frontend/coprs_frontend/coprs/forms.py

```diff
@@ -227,10 +227,16 @@ def is_url(self, url):
         #  copr://username/projectname
         #  ^^ schema ^^ netlock  ^^ path
         if parsed.scheme == "copr":
+            ownername = parsed.netloc
             # check if projectname missed
             path_split = parsed.path.split("/")
+            coprname = path_split[1]
             if len(path_split) < 2 or path_split[1] == "":
                 return False
```

Comment:
My apologies @jaitjacob, I just realized one more problem. The `coprname = path_split[1]` line should be here, after the condition, not before it. Because otherwise I can crash it with the following input: `copr://foo`

Reply:
nice catch, fixed now